d2l/tensorflow.py

```diff
@@ -484,6 +484,7 @@ def train_ch6(net_fn, train_iter, test_iter, num_epochs, lr,
     callback = TrainCallback(net, train_iter, test_iter, num_epochs,
                              device_name)
     net.fit(train_iter, epochs=num_epochs, verbose=0, callbacks=[callback])
+    return net
```

Comment:
a question about `return net`. when it's needed? I found removing it works fine

Reply:
There's one example here https://github.com/d2l-ai/d2l-en/pull/1054/files#diff-cf23455381e755a4d0f02d266b186b6eR552 requires accessing the trained layers/params in the model object. Since we the model is created inside the function, we will have to return it here so we can use it later.

Reply:
got it. thanks. the issue it causes is that jupyter will print the returned network, which is not necessary in most cases